src/components/link/Link.block.ts

```diff
@@ -0,0 +1,23 @@
+import { Block } from "../../modules/Block/Block";
+import { Router } from "../../modules/Router/Router";
+interface LinkProps {
+  router: Router;
+  to: string;
+}
+export class Link extends Block<any> {
+  constructor({ to, router }: LinkProps) {
```

Comment:
```suggestion
export class Link extends Block<LinkProps> {
  constructor({ to, router }: LinkProps) {
```
Стоит типизировать пропсы так, тогда можно будет пользоваться подсказками в методах компонента
<img width="638" alt="Screenshot 2021-10-31 at 17 10 28" src="https://user-images.githubusercontent.com/28347933/159214212-01167f40-8ed8-4212-b191-f1fb21fe1dfe.png">